common/scala/src/main/scala/whisk/core/entity/WhiskActivation.scala

```diff
@@ -113,6 +113,6 @@ object WhiskActivation
     override val collectionName = "activations"
     override implicit val serdes = jsonFormat13(WhiskActivation.apply)
 
-    override val cacheEnabled = true
+    override val cacheEnabled = false
```

Comment:
add a comment perhaps that caching activations is not of much use in the common case?